srcs/http/http_exception.hpp

```diff
@@ -0,0 +1,21 @@
+#ifndef HTTP_RESPONSE_EXCEPTION_HPP_
+#define HTTP_RESPONSE_EXCEPTION_HPP_
```

Comment:
細かいですがここからも response いらなさそうです
```suggestion
#ifndef HTTP_EXCEPTION_HPP_
#define HTTP_EXCEPTION_HPP_
```